taca/backup/backup.py

```diff
@@ -257,39 +276,61 @@ def _clean_tmp_files(self, files):
-            logger.warn(f'Not able to log "pdc_archived" timestamp for run {run}')
+        if "AV" not in run:
+            try:
+                run_vals = run.split("_")
+                if len(run_vals[0]) == 8:
+                    run_date = run_vals[0][2:]
+                else:
+                    run_date = run_vals[0]
+                run_fc = f"{run_date}_{run_vals[-1]}"
+                couch_connection = statusdb.StatusdbSession(self.couch_info).connection
+                db = couch_connection[self.couch_info["db"]]
+                fc_names = {e.key: e.id for e in db.view("names/name", reduce=False)}
+                d_id = fc_names[run_fc]
+                doc = db.get(d_id)
+                doc["pdc_archived"] = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+                db.save(doc)
+                logger.info(
+                    f'Logged "pdc_archived" timestamp for fc {run} in statusdb doc "{d_id}"'
+                )
+            except:
+                logger.warning(
+                    f'Not able to log "pdc_archived" timestamp for run {run}'
+                )
+        else:
+            try:
+                element_db_connection = statusdb.ElementRunsConnection(
+                    self.CONFIG.get("statusdb", {}), dbname="element_runs"
+                )
+                run_doc = element_db_connection.get_db_entry(run)
+                run_doc["pdc_archived"] = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+                element_db_connection.upload_to_statusdb(run_doc)
+            except:
+                logger.warning(
+                    f'Not able to log "pdc_archived" timestamp for run {run}'
+                )
 
     def _is_ready_to_archive(self, run, ext):
-        """Check if the run to be encrypted has finished sequencing and has been copied completely to nas"""
+        """Check if the run to be encrypted has finished sequencing and has been copied completely to ngi_data"""
         archive_ready = False
         run_path = run.abs_path
         rta_file = os.path.join(run_path, self.finished_run_indicator)
         cp_file = os.path.join(run_path, self.copy_complete_indicator)
         if (
-            os.path.exists(rta_file)
-            and os.path.exists(cp_file)
-            and (not self.file_in_pdc(run.tar_encrypted))
-        ) or (
-            self._get_run_type(run.name) in ["promethion", "minion"]
-            and os.path.exists(os.path.join(run_path, ".sync_finished"))
+            (
+                os.path.exists(rta_file)
+                and os.path.exists(cp_file)
+                and (not self.file_in_pdc(run.tar_encrypted))
+            )
+            or (
+                self._get_run_type(run.name) in ["promethion", "minion"]
+                and os.path.exists(os.path.join(run_path, ".sync_finished"))
+            )
+            or (
+                self._get_run_type(run.name) == "aviti"
+                and os.path.exists(os.path.join(run_path, "RunUploaded.json"))
```

Comment:
So this will start archiving as soon as sequencing has finished? I guess that's what we want

Reply:
Well, the run should be in nosync, which is once it's been demuxed and transferred to miarka. The only catch is that if we need to redemux, we have to wait for the archiving to finish.